README.md

```diff
@@ -123,7 +123,22 @@ Run integration tests in virtualenvs using tox:
 export REPORTS_DIR=<path to the folder where JUnit reports will be stored>
 tox -e py27-integration,py35-integration,py36-integration,py37-integration -- samples/network_object.yml -i inventory/sample_hosts
 ```
-
+### Running style check locally
+Install [Flake8](http://flake8.pycqa.org/en/latest/) locally:
+```
+pip install flake8
+```
+Flake8 configuration is defined in the `tox.ini` file :
+```
+[flake8] 
```

Comment:
Let's not duplicate `flake8` config in the `README.md` file. Just put a fink to the `tox.ini` file instead.